### ISSUE TITLE
Should "mutt -Q folder" output differ from upstream?

### ISSUE THREAD

**Issue by scottkosty** [neomutt/neomutt#1706]
In a recent upstream mutt, I get the following:

```
$ mutt -Q "folder"
folder="~/.mail"
$
``` 

neomutt now gives the following:

```
$ mutt -Q "folder"
set folder = "/home/scott/.mail"
$
``` 

There are two differences, the "set", and also the expanding of `~`. Are these differences expected? They don't bother me. I was just surprised so thought I would make sure that it's expected.

**Comment by flatcap** [comment]
The change **is** intentional.

When I rewrote the config-handling code, I wrote two versions of the dump code:
- **mutt** -- the 'legacy' format, e.g. `sidebar_visible is set`
- **neomutt** -- valid config, e.g. `set sidebar_visible = yes`

The neomutt code has other features too (not yet implemented as command line options):
- Show the default values
- Only show config that has been changed by the user
- Show disabled config
